src/PhpSpreadsheet/Chart/DataSeriesValues.php

```diff
@@ -224,14 +224,22 @@ public function getFillColor()
     /**
      * Set fill color for series.
      *
-     * @param string $color HEX color
+     * @param array|string $color HEX color or array with HEX colors
      *
      * @return   DataSeriesValues
      */
     public function setFillColor($color)
     {
-        if (!preg_match('/^[a-f0-9]{6}$/i', $color)) {
-            throw new Exception('Invalid hex color for chart series');
+        if (is_array($color)) {
+            foreach ($color as $colorValue) {
+                if (!preg_match('/^[a-f0-9]{6}$/i', $colorValue)) {
+                    throw new Exception(sprintf('Invalid hex color for chart series (color: "%s")', $colorValue));
+                }
+            }
+        } else {
+            if (!preg_match('/^[a-f0-9]{6}$/i', $color)) {
```

Comment:
Avoid duplicating validation code by creating a private method like "validateColor(string $color): void" that throw exception if need be.